### ISSUE TITLE
error: Could not open codec [av1_vaapi]: Generic error in an external library

### ISSUE THREAD

**Issue by tlivolsi** [LizardByte/Sunshine#1757]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Sunshine fails during its test for an AV1 encoder.

### Additional Context

libva info: va_openDriver() returns 0
vainfo: VA-API version: 1.14 (libva 2.12.0)
vainfo: Driver version: Mesa Gallium driver 23.2.1 - kisak-mesa PPA for GFX1100 (gfx1100, LLVM 15.0.7, DRM 3.54, 6.5.6-060506-generic)
vainfo: Supported profile and entrypoints
      VAProfileH264ConstrainedBaseline:	VAEntrypointVLD
      VAProfileH264ConstrainedBaseline:	VAEntrypointEncSlice
      VAProfileH264Main               :	VAEntrypointVLD
      VAProfileH264Main               :	VAEntrypointEncSlice
      VAProfileH264High               :	VAEntrypointVLD
      VAProfileH264High               :	VAEntrypointEncSlice
      VAProfileHEVCMain               :	VAEntrypointVLD
      VAProfileHEVCMain               :	VAEntrypointEncSlice
      VAProfileHEVCMain10             :	VAEntrypointVLD
      VAProfileHEVCMain10             :	VAEntrypointEncSlice
      VAProfileJPEGBaseline           :	VAEntrypointVLD
      VAProfileVP9Profile0            :	VAEntrypointVLD
      VAProfileVP9Profile2            :	VAEntrypointVLD
      VAProfileAV1Profile0            :	VAEntrypointVLD
      VAProfileAV1Profile0            :	VAEntrypointEncSlice
      VAProfileNone                   :	VAEntrypointVideoProc

### Host Operating System

Linux

### Operating System Version

Linux Mint 21.2

### Architecture

64 bit

### Sunshine commit or version

0.21.0

### Package

Linux - deb

### GPU Type

AMD

### GPU Model

AMD 7900 XTX

### GPU Driver/Mesa Version

Mesa 23.2.1

### Config

```Shell
fec_percentage = 30
qp = 14
min_threads = 4
min_log_level = 1
```

### Relevant log output

```shell
[2023:10:16:09:52:15]: Info: Sunshine version: v0.21.0
[2023:10:16:09:52:15]: Info: System tray created
[2023:10:16:09:52:15]: Error: Couldn't initialize cuda: CUDA_ERROR_NO_DEVICE:no CUDA-capable device is detected
[2023:10:16:09:52:15]: Error: Environment variable WAYLAND_DISPLAY has not been defined
[2023:10:16:09:52:15]: Info: Detecting monitors
[2023:10:16:09:52:15]: Info: Detected monitor 0: DisplayPort-0, connected: false
[2023:10:16:09:52:15]: Info: Detected monitor 1: DisplayPort-1, connected: false
[2023:10:16:09:52:15]: Info: Detected monitor 2: HDMI-A-0, connected: true
[2023:10:16:09:52:15]: Info: Detected monitor 3: HDMI-A-1, connected: false
[2023:10:16:09:52:15]: Info: // Testing for available encoders, this may generate errors. You can safely ignore those errors. //
[2023:10:16:09:52:15]: Info: Trying encoder [nvenc]
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Error: [AVHWDeviceContext @ 0x55de29c53700] cu->cuInit(0) failed
[2023:10:16:09:52:15]: Error:  -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2023:10:16:09:52:15]: Error: 
[2023:10:16:09:52:15]: Error: Failed to create a CUDA device: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Error: [AVHWDeviceContext @ 0x55de29a37380] cu->cuInit(0) failed
[2023:10:16:09:52:15]: Error:  -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2023:10:16:09:52:15]: Error: 
[2023:10:16:09:52:15]: Error: Failed to create a CUDA device: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Error: [AVHWDeviceContext @ 0x55de29b43300] cu->cuInit(0) failed
[2023:10:16:09:52:15]: Error:  -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2023:10:16:09:52:15]: Error: 
[2023:10:16:09:52:15]: Error: Failed to create a CUDA device: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Error: [AVHWDeviceContext @ 0x55de29c6d080] cu->cuInit(0) failed
[2023:10:16:09:52:15]: Error:  -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2023:10:16:09:52:15]: Error: 
[2023:10:16:09:52:15]: Error: Failed to create a CUDA device: Generic error in an external library
[2023:10:16:09:52:15]: Info: Encoder [nvenc] failed
[2023:10:16:09:52:15]: Info: Trying encoder [vaapi]
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Warning: [h264_vaapi @ 0x55de29b34600] Driver does not support some wanted packed headers (wanted 0xd, found 0).
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Warning: [hevc_vaapi @ 0x55de29cf9380] Driver does not support some wanted packed headers (wanted 0xd, found 0x1).
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9880] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9880] Attribute type:52 is not supported.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9880] Attribute type:53 is not supported.
[2023:10:16:09:52:15]: Error: [av1_vaapi @ 0x55de29af9880] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:15]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de2a594540] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de2a594540] Attribute type:52 is not supported.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de2a594540] Attribute type:53 is not supported.
[2023:10:16:09:52:15]: Error: [av1_vaapi @ 0x55de2a594540] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:15]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29b67780] No quality level set; using default (25).
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29b67780] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29b67780] Attribute type:52 is not supported.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29b67780] Attribute type:53 is not supported.
[2023:10:16:09:52:15]: Error: [av1_vaapi @ 0x55de29b67780] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:15]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:15]: Info: Color depth: 8-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9540] No quality level set; using default (25).
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9540] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9540] Attribute type:52 is not supported.
[2023:10:16:09:52:15]: Warning: [av1_vaapi @ 0x55de29af9540] Attribute type:53 is not supported.
[2023:10:16:09:52:15]: Error: [av1_vaapi @ 0x55de29af9540] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:15]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:15]: Info: Screencasting with KMS
[2023:10:16:09:52:15]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:15]: Info: SDR color coding [Rec. 709]
[2023:10:16:09:52:15]: Info: Color depth: 10-bit
[2023:10:16:09:52:15]: Info: Color range: [JPEG]
[2023:10:16:09:52:15]: Error: [AVHWFramesContext @ 0x55de29ce2dc0] Unsupported format: yuv420p10le.
[2023:10:16:09:52:15]: Warning: vaapi: hevc missing sps->vui parameters
[2023:10:16:09:52:15]: Info: 
[2023:10:16:09:52:15]: Info: // Ignore any errors mentioned above, they are not relevant. //
[2023:10:16:09:52:15]: Info: 
[2023:10:16:09:52:15]: Info: Found H.264 encoder: h264_vaapi [vaapi]
[2023:10:16:09:52:15]: Info: Found HEVC encoder: hevc_vaapi [vaapi]
[2023:10:16:09:52:15]: Info: Adding avahi service Sunshine
[2023:10:16:09:52:15]: Info: Configuration UI available at [https://localhost:47990]
[2023:10:16:09:52:16]: Info: Avahi service name collision, renaming service to Sunshine #2
[2023:10:16:09:52:16]: Info: Adding avahi service Sunshine #2
[2023:10:16:09:52:17]: Info: Avahi service name collision, renaming service to Sunshine #3
[2023:10:16:09:52:17]: Info: Adding avahi service Sunshine #3
[2023:10:16:09:52:18]: Info: Avahi service Sunshine #3 successfully established.
[2023:10:16:09:52:35]: Info: // Testing for available encoders, this may generate errors. You can safely ignore those errors. //
[2023:10:16:09:52:35]: Info: Trying encoder [nvenc]
[2023:10:16:09:52:35]: Info: Screencasting with KMS
[2023:10:16:09:52:35]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:35]: Info: Screencasting with KMS
[2023:10:16:09:52:35]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:35]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:35]: Info: Color depth: 8-bit
[2023:10:16:09:52:35]: Info: Color range: [JPEG]
[2023:10:16:09:52:35]: Error: [AVHWDeviceContext @ 0x7f71681089c0] cu->cuInit(0) failed
[2023:10:16:09:52:35]: Error:  -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2023:10:16:09:52:35]: Error: 
[2023:10:16:09:52:35]: Error: Failed to create a CUDA device: Generic error in an external library
[2023:10:16:09:52:35]: Info: Screencasting with KMS
[2023:10:16:09:52:35]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:35]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:35]: Info: Color depth: 8-bit
[2023:10:16:09:52:35]: Info: Color range: [JPEG]
[2023:10:16:09:52:35]: Error: [AVHWDeviceContext @ 0x7f7168099e40] cu->cuInit(0) failed
[2023:10:16:09:52:35]: Error:  -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2023:10:16:09:52:35]: Error: 
[2023:10:16:09:52:35]: Error: Failed to create a CUDA device: Generic error in an external library
[2023:10:16:09:52:35]: Info: Encoder [nvenc] failed
[2023:10:16:09:52:35]: Info: Trying encoder [vaapi]
[2023:10:16:09:52:35]: Info: Screencasting with KMS
[2023:10:16:09:52:35]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:35]: Info: Screencasting with KMS
[2023:10:16:09:52:35]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:35]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:35]: Info: Color depth: 8-bit
[2023:10:16:09:52:35]: Info: Color range: [JPEG]
[2023:10:16:09:52:35]: Warning: [h264_vaapi @ 0x7f7168213700] Driver does not support some wanted packed headers (wanted 0xd, found 0).
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:36]: Info: Color depth: 8-bit
[2023:10:16:09:52:36]: Info: Color range: [JPEG]
[2023:10:16:09:52:36]: Warning: [hevc_vaapi @ 0x7f716802f700] Driver does not support some wanted packed headers (wanted 0xd, found 0x1).
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:36]: Info: Color depth: 8-bit
[2023:10:16:09:52:36]: Info: Color range: [JPEG]
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71682c0e40] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71682c0e40] Attribute type:52 is not supported.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71682c0e40] Attribute type:53 is not supported.
[2023:10:16:09:52:36]: Error: [av1_vaapi @ 0x7f71682c0e40] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:36]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:36]: Info: Color depth: 8-bit
[2023:10:16:09:52:36]: Info: Color range: [JPEG]
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b7c80] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b7c80] Attribute type:52 is not supported.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b7c80] Attribute type:53 is not supported.
[2023:10:16:09:52:36]: Error: [av1_vaapi @ 0x7f71680b7c80] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:36]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:36]: Info: Color depth: 8-bit
[2023:10:16:09:52:36]: Info: Color range: [JPEG]
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b46c0] No quality level set; using default (25).
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b46c0] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b46c0] Attribute type:52 is not supported.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b46c0] Attribute type:53 is not supported.
[2023:10:16:09:52:36]: Error: [av1_vaapi @ 0x7f71680b46c0] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:36]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:36]: Info: Color depth: 8-bit
[2023:10:16:09:52:36]: Info: Color range: [JPEG]
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b23c0] No quality level set; using default (25).
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b23c0] Multiple slices were requested but this codec does not support controlling slices.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b23c0] Attribute type:52 is not supported.
[2023:10:16:09:52:36]: Warning: [av1_vaapi @ 0x7f71680b23c0] Attribute type:53 is not supported.
[2023:10:16:09:52:36]: Error: [av1_vaapi @ 0x7f71680b23c0] Failed to query config attribute: 0 (success (no error)).
[2023:10:16:09:52:36]: Error: Could not open codec [av1_vaapi]: Generic error in an external library
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 709]
[2023:10:16:09:52:36]: Info: Color depth: 10-bit
[2023:10:16:09:52:36]: Info: Color range: [JPEG]
[2023:10:16:09:52:36]: Error: [AVHWFramesContext @ 0x7f7168305fc0] Unsupported format: yuv420p10le.
[2023:10:16:09:52:36]: Warning: vaapi: hevc missing sps->vui parameters
[2023:10:16:09:52:36]: Info: 
[2023:10:16:09:52:36]: Info: // Ignore any errors mentioned above, they are not relevant. //
[2023:10:16:09:52:36]: Info: 
[2023:10:16:09:52:36]: Info: Found H.264 encoder: h264_vaapi [vaapi]
[2023:10:16:09:52:36]: Info: Found HEVC encoder: hevc_vaapi [vaapi]
[2023:10:16:09:52:36]: Info: Executing [Desktop]
[2023:10:16:09:52:36]: Info: CLIENT CONNECTED
[2023:10:16:09:52:36]: Info: Screencasting with KMS
[2023:10:16:09:52:36]: Info: Found monitor for DRM screencasting
[2023:10:16:09:52:36]: Info: SDR color coding [Rec. 601]
[2023:10:16:09:52:36]: Info: Color depth: 8-bit
[2023:10:16:09:52:36]: Info: Color range: [MPEG]
[2023:10:16:09:52:36]: Warning: [hevc_vaapi @ 0x7f71541e4780] Driver does not support some wanted packed headers (wanted 0xd, found 0x1).
```

**Comment by d-air1** [comment]
I have an nvidia rtx 3080 and upon upgrading to this version I also get this error in a big red banner saying it could not find av1 encoder. However unlike the OP, to my knowledge the rtx 3080 does not support av1 encoding, so I don't know why this is being shown to me in the first place. 

Here are my logs
```
[2023:10:16:18:44:43]: Info: Sunshine version: v0.21.0
[2023:10:16:18:44:43]: Warning: Couldn't find app image at path [/var/lib/flatpak/app/com.valvesoftware.Steam/x86_64/stable/a4c6945dc1cb48472fae369277a4350c9592c5d26239243bd06afc183da3dd7c/export/share/icons/hicolor/256x256/apps/com.valvesoftware.Steam.png]
[2023:10:16:18:44:43]: Info: System tray created
[2023:10:16:18:44:43]: Error: Failed to create session: This hardware does not support NvFBC
[2023:10:16:18:44:43]: Error: Failed to gain CAP_SYS_ADMIN
[2023:10:16:18:44:43]: Error: Environment variable WAYLAND_DISPLAY has not been defined
[2023:10:16:18:44:43]: Info: Detecting monitors
[2023:10:16:18:44:43]: Info: Detected monitor 0: DP-0, connected: false
[2023:10:16:18:44:43]: Info: Detected monitor 1: DP-1, connected: false
[2023:10:16:18:44:43]: Info: Detected monitor 2: HDMI-0, connected: false
[2023:10:16:18:44:43]: Info: Detected monitor 3: DP-2, connected: true
[2023:10:16:18:44:43]: Info: Detected monitor 4: DP-3, connected: false
[2023:10:16:18:44:43]: Info: Detected monitor 5: DP-4, connected: true
[2023:10:16:18:44:43]: Info: Detected monitor 6: DP-5, connected: false
[2023:10:16:18:44:43]: Info: Detected monitor 7: HDMI-1, connected: false
[2023:10:16:18:44:43]: Info: // Testing for available encoders, this may generate errors. You can safely ignore those errors. //
[2023:10:16:18:44:43]: Info: Trying encoder [nvenc]
[2023:10:16:18:44:43]: Info: Screencasting with X11
[2023:10:16:18:44:43]: Info: Screencasting with X11
[2023:10:16:18:44:43]: Info: SDR color coding [Rec. 601]
[2023:10:16:18:44:43]: Info: Color depth: 8-bit
[2023:10:16:18:44:43]: Info: Color range: [JPEG]
[2023:10:16:18:44:43]: Info: Screencasting with X11
[2023:10:16:18:44:43]: Info: SDR color coding [Rec. 601]
[2023:10:16:18:44:43]: Info: Color depth: 8-bit
[2023:10:16:18:44:43]: Info: Color range: [JPEG]
[2023:10:16:18:44:43]: Info: Screencasting with X11
[2023:10:16:18:44:43]: Info: SDR color coding [Rec. 601]
[2023:10:16:18:44:43]: Info: Color depth: 8-bit
[2023:10:16:18:44:43]: Info: Color range: [JPEG]
[2023:10:16:18:44:43]: Warning: [av1_nvenc @ 0x55cf95c437c0] Codec not supported
[2023:10:16:18:44:43]: Fatal: [av1_nvenc @ 0x55cf95c437c0] Provided device doesn't support required NVENC features
[2023:10:16:18:44:43]: Error: Could not open codec [av1_nvenc]: Function not implemented
[2023:10:16:18:44:43]: Info: Screencasting with X11
[2023:10:16:18:44:43]: Info: SDR color coding [Rec. 601]
[2023:10:16:18:44:43]: Info: Color depth: 8-bit
[2023:10:16:18:44:43]: Info: Color range: [JPEG]
[2023:10:16:18:44:43]: Warning: [av1_nvenc @ 0x55cf95c437c0] Codec not supported
[2023:10:16:18:44:43]: Fatal: [av1_nvenc @ 0x55cf95c437c0] Provided device doesn't support required NVENC features
[2023:10:16:18:44:43]: Error: Could not open codec [av1_nvenc]: Function not implemented
[2023:10:16:18:44:43]: Info: Screencasting with X11
[2023:10:16:18:44:43]: Info: SDR color coding [Rec. 709]
[2023:10:16:18:44:43]: Info: Color depth: 10-bit
[2023:10:16:18:44:43]: Info: Color range: [JPEG]
[2023:10:16:18:44:43]: Error: cuda::cuda_t doesn't support any format other than AV_PIX_FMT_NV12
[2023:10:16:18:44:43]: Info: 
[2023:10:16:18:44:43]: Info: // Ignore any errors mentioned above, they are not relevant. //
[2023:10:16:18:44:43]: Info: 
[2023:10:16:18:44:43]: Info: Found H.264 encoder: h264_nvenc [nvenc]
[2023:10:16:18:44:43]: Info: Found HEVC encoder: hevc_nvenc [nvenc]
[2023:10:16:18:44:43]: Info: Adding avahi service Sunshine
[2023:10:16:18:44:43]: Info: Configuration UI available at [https://localhost:47990]
[2023:10:16:18:44:44]: Info: Avahi service Sunshine successfully established.
[2023:10:16:18:44:51]: Warning: Couldn't discover any IPv4 UPNP devices
[2023:10:16:18:45:45]: Info: Web UI: [127.0.0.1] -- not authorized
[2023:10:16:18:46:59]: Warning: Couldn't discover any IPv4 UPNP devices
[2023:10:16:18:49:07]: Warning: Couldn't discover any IPv4 UPNP devices
[2023:10:16:18:51:15]: Warning: Couldn't discover any IPv4 UPNP devices
[2023:10:16:18:53:23]: Warning: Couldn't discover any IPv4 UPNP devices
```

**Comment by cgutman** [comment]
> However unlike the OP, to my knowledge the rtx 3080 does not support av1 encoding, so I don't know why this is being shown to me in the first place.

Yeah, that's unfortunate. We probably shouldn't promote FFmpeg fatal "errors" like that to Fatal status to avoid them showing up there. It's not related to OP's issue though.

**Comment by ehabajmal** [comment]
> I have an nvidia rtx 3080 and upon upgrading to this version I also get this error in a big red banner saying it could not find av1 encoder. However unlike the OP, to my knowledge the rtx 3080 does not support av1 encoding, so I don't know why this is being shown to me in the first place.
Similarly, I have an RTX 3070 and get the same exact error when using the new 0.21.0 release. 0.20.0 doesn't show this error.
![image](https://github.com/LizardByte/Sunshine/assets/33014707/1e47b26e-639f-48e7-99cf-71324258a14f)

**Comment by tlivolsi** [comment]
If a developer sees this, let me know if you'd like debugging logs or anything else. I'm happy to help fix this, presuming the problem isn't something quirky on my end.

**Comment by tlivolsi** [comment]
This is resolved in Mesa 23.3.
